egs/swbd/s5c/local/chain/run_tdnn_6z.sh

```diff
@@ -45,6 +45,7 @@ pool_window=
 pool_type='none'
```

Comment:
Instead of making the changes to 6z, please copy-and-modify to a new letter (7d or whatever is currently free)- then you can put the appropriate comparisons obtained from compare_wer.sh, in the script.  You can also update the README.txt to point to the new number as the current recommended recipe.